docs/notebooks/libs/utils.py

```diff
@@ -0,0 +1,86 @@
+import re
```

Comment:
Why are these packages not imported centrally?

Reply:
see 
1ea4625

Reply:
ok, thanks.